api/src/test/java/io/strimzi/api/kafka/model/KafkaConnectorTest.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2018, Strimzi authors.
```

Comment:
2019

Reply:
@ppatierno about this https://github.com/strimzi/strimzi-kafka-operator/pull/2073#discussion_r335313313 (sorry I could not answer in the same thread) you mean something like this description?
`    @Description("The Kafka Connect configuration. Properties with the following prefixes cannot be set: " + FORBIDDEN_PREFIXES)
    public Map<String, Object> getConfig() {
        return config;
    }`

Reply:
@pamelipluas yes, even for `Kafka` we have some forbidden configuration parameters which are set internally and the user cannot set in the related CR.